operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -1236,35 +1247,49 @@ spec:
                     - name: STRIMZI_OPERATION_TIMEOUT_MS
                       value: "300000"
                     - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
-                      value: registry.redhat.io/amq-streams/kafka-37-rhel9@sha256:42bf60ce31540dd61fab2c9886d791e41f063ea6f86628694b9e60e49bc8951b
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.annotations['kafka-current']
```

Comment:
Shouldn't this say `metadata.annotations['kafka-current-image']`?

Reply:
Yes, thanks for the catch, I missed this